src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -67,6 +67,26 @@ public void CanGetAllTimesAfterCreation()
             ValidateSetTimes(item, beforeTime, afterTime);
         }
 
+        [Fact]
+        public void TimesIncludeMillisecondPart()
+        {
+            T item = GetExistingItem();
+            Assert.All(TimeFunctions(), (function) =>
+            {
+                var msec = 0;
+                for (int i = 0; i < 3; i++)
+                {
+                    msec = function.Getter(item).Millisecond;
+                    if (msec != 0)
```

Comment:
Are there common file system with seconds resolution for timestamps? This test will fail on them ... I guess we will find out.

Reply:
I figured we will...

Reply:
FAT has 2 second granularity.